type-metadata/src/tests.rs

```diff
@@ -22,14 +22,6 @@ fn primitives_metadata_impl_should_work() {
 	assert_eq!(<&str>::type_ident(), IdentKind::Str);
 	assert_eq!(i8::type_ident(), IdentKind::I8);
 
-	assert_eq!(
```

Comment:
~~Why remove those tests for `Option` and `Result`? They now make up for awesome test cases for custom test types without type def.~~

Nvm, but I cannot delete this somehow.